### PR TITLE
Release candidate 2.3.3

### PR DESCRIPTION
**Pull request by amirhosv**
*Description of changes:*

+ Starting from this release, artifacts for MacOS AARCH64 are vended.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/gradle-kt-dsl/lib/build.gradle.kts [hunk 1]
```diff
@@ -31,7 +32,7 @@ dependencies {
 
     // for OSX-x86_64, we do not have FIPS artifacts
```
Comment (WillChilds-Klein) [inline examples/gradle-kt-dsl/lib/build.gradle.kts:33]:
```suggestion
    // for OSX, we do not have FIPS artifacts
```

**Review by WillChilds-Klein — APPROVED** [review]

File: examples/gradle-kt-dsl/lib/build.gradle.kts (hunk 1 above)
Comment (amirhosv) [reply examples/gradle-kt-dsl/lib/build.gradle.kts]: How fast comments go out-of-sync :(

**Review by geedo0 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -146,9 +153,12 @@ The artifactId for FIPS builds is `AmazonCorrettoCryptoProvider-FIPS`.
 
 The classifier attribute could be set to `linux-aarch_64` to use ACCP on Linux ARM64 platforms.
 
-Starting from verions 2.3.2, the classifier attribute could be set to `osx-x86_64` to use ACCP
+Starting from version 2.3.2, the classifier attribute could be set to `osx-x86_64` to use ACCP
 on MacOS X86 platforms. The FIPS artifacts do not support MacOS X86.
 
+Starting from version 2.3.3, the classifier attribute could be set to `osx-aarch_64` to use ACCP
+on MacOS AARCH64 (a.k.a. Arm64) platforms. The FIPS artifacts do not support MacOS AARCH64.
```
[the remark this answers is not in the document]
Comment (amirhosv) [reply README.md]: Done.

File: examples/gradle-kt-dsl/lib/build.gradle.kts [hunk 3]
```diff
@@ -31,7 +32,7 @@ dependencies {
...
     val accpArtifactId =
-        if (project.hasProperty("fips") && osdetector.classifier != "osx-x86_64")
+        if (project.hasProperty("fips") && osdetector.classifier !in PLATFORMS_WITHOUT_FIPS_SUPPORT)
```
[the remark this answers is not in the document]
Comment (amirhosv) [reply examples/gradle-kt-dsl/lib/build.gradle.kts:35]: My next big JVM based project will be in Kotlin. It provides a lot of operators that can be overloaded for user defined types too: https://kotlinlang.org/docs/operator-overloading.html

**Review by WillChilds-Klein — APPROVED** [review]

**Review by brian-jarvis-aws — APPROVED** [review]

**Pull request merged**